media/css/products/vpn/components/block.scss

```diff
@@ -0,0 +1,371 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../../protocol/css/includes/lib';
+@import '../includes/lib';
+
+//* -------------------------------------------------------------------------- */
+// Content Block
+
+.vpn-content-block {
+    @include flexbox;
+    @include align-items(center);
+    -moz-outline-radius: $border-radius-sm;
+    border-radius: $border-radius-sm;
+    color: $color-black;
+    margin-bottom: $layout-lg;
+    outline: 4px solid $color-black;
```

Comment:
Should this be a protocol token?

Reply:
Hmm, I'm not sure we have any specic tokens for border sizing/styling in Protocol?

Reply:
Ahh – I was thinking about `$border-radius-xs`… but that's for `border-radius`. Please disregard!